message.go

```diff
@@ -60,7 +62,11 @@ type Decoder struct {
 
 // NewDecoder returns a decoder for the given reader.
 func NewDecoder(r io.Reader) Decoder {
-	return Decoder{r: r}
+	return Decoder{
+		// The packet reader solves the issue of reading packets at once (required for UDP) while
+		// still being able to read byte by byte to verify the input.
+		r: bufio.NewPacketReader(r, 1500),
```

Comment:
Maybe this could be in a later PR, but it would likely make sense to make this number configurable, and related to the packet size that we expect to read (which I believe is already configurable? not sure)

Reply:
I didn't understand the implementation of the buffer size TBH. I think it just wasn't implemented entirely. While I played a bit with different buffer sizes, the client just does not understand a message with a significantly low buffer size, as the message will just be cropped. And even choosing a value here that makes more sense for the messages we senc, the message and the client aren't really coupled, so we could easily break things, when we decide to send larger messages.

So I would keep it for now, until we update the buffer size in general.

WDYT?

Reply:
Sure, I'll open a ticket for it then 👍

Reply:
https://github.com/nitrado/connqc/issues/22